app/main/routes.py

```diff
@@ -659,7 +661,9 @@ def download_record(record_id: uuid.UUID):
             "Content-Disposition": "attachment;filename=" + download_filename
         },
     )
-
+    current_app.logger.info(
```

Comment:
lets add a test to show we log this info

Reply:
caplog used to capture logging details for test cases in test_download.py